sqlbrite-lint/src/test/java/com/squareup/sqlbrite3/SqlBriteArgCountDetectorTest.kt

```diff
@@ -65,11 +63,14 @@ class SqlBriteArgCountDetectorTest : LintDetectorTest() {
 
               }
             """.trimIndent()))
+        .detector(SqlBriteArgCountDetector())
```

Comment:
no need for passing the detctor as the issues function takes the Issue which has an implementation that links to SqlBriteArgCountDetector `Implementation(SqlBriteArgCountDetector::class.java, EnumSet.of(JAVA_FILE, TEST_SOURCES)))`